src/awx_plugins/credentials/github_app.py

```diff
@@ -132,28 +133,48 @@ class EmptyKwargs(TypedDict):
 
 GH_CLIENT_ID_TRAILER_LENGTH = 16
 HEXADECIMAL_BASE = 16
+GH_CLIENT_ID_MIN_LENGTH = 19
+GH_CLIENT_ID_MAX_LENGTH = 22
+GH_CLIENT_ID_LENGTH_BOUNDS = range(
+    GH_CLIENT_ID_MIN_LENGTH,
+    GH_CLIENT_ID_MAX_LENGTH + 1,
+)  # min 19, max 22
```

Comment:
Since you've reintroduced explicit constants, this comment is no longer necessary. In fact, there is harm in it potentially going out of sync and becoming confusing.
```suggestion
)
```